config/config.go

```diff
@@ -268,6 +264,11 @@ type Config struct {
 	EnableBatchDML bool   `toml:"enable-batch-dml" json:"enable-batch-dml"`
 	MemQuotaQuery  int64  `toml:"mem-quota-query" json:"mem-quota-query"`
 	OOMAction      string `toml:"oom-action" json:"oom-action"`
+
+	// CheckMb4ValueInUTF8, EnableCollectExecutionInfo, Plugin unused since bootstrap v92
+	CheckMb4ValueInUTF8        AtomicBool `toml:"check-mb4-value-in-utf8" json:"check-mb4-value-in-utf8"`
+	EnableCollectExecutionInfo bool       `toml:"enable-collect-execution-info" json:"enable-collect-execution-info"`
+	Plugin                     Plugin     `toml:"plugin" json:"plugin"`
```

Comment:
I like the idea of moving to the bottom, but maybe say they are deprecated and not removed. They are different to config options that were converted to sysvars, because they still take full effect.

What I meant though is that they should not be visible in `SHOW CONFIG` or `select @@tidb_config`. You can do this by modifying either `removedConfig` (adding them here) or modifying `RemovedVariableCheck()` and `ContainHiddenConfig()` to also strip out the old locations for instance variables.